dev/src/cubegen.cpp

```diff
@@ -821,7 +832,27 @@ nfr("cg_load_vox", "name,material_palette", "SI?", "R:voxels]S?",
                 }
             }
             if (voxvec->SLen() < (ssize_t)node_to_model.size()) {
-                return errf(".vox file uses object deduplication feature that is not supported\n");
+                auto new_voxvec = vm.NewVec(0, 0, TYPE_ELEM_VECTOR_OF_RESOURCE);
```

Comment:
doesn't matter too much, but more elegant to just call `voxvec->Push` which will enlarge the existing vector?

Reply:
I think the problem may be that the new voxels aren't necessarily at the end, so I'd need to insert into the middle of the vector

Reply:
There's an `Insert` too :)

Reply:
Oh heh, right! Well, I guess theoretically the vox file could even have the shape indexes out of order too, but I doubt that ever happens in practice. I think this code gets a bit clumsier regardless, since you would need to copy from the vec as it's being modified